src/web/core/src/main/java/org/geoserver/web/data/layer/LayerPage.java

```diff
@@ -76,10 +77,12 @@ protected Component getComponentForProperty(String id, IModel<LayerInfo> itemMod
                     return f;
                 } else if(property == SRS) {
                     return new Label(id, SRS.getModel(itemModel));
+                } else if (property == TITLE) {
```

Comment:
The `if (property == WORKSPACE)` segment above is probably invalid now if you are no longer displaying workspace?